<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="FluidFlowView" name="viewWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModes">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionView"/>
    <addaction name="actionEdit"/>
    <addaction name="separator"/>
    <addaction name="actionColourBoundary"/>
    <addaction name="actionColourValue"/>
    <addaction name="separator"/>
    <addaction name="actionShowMesh"/>
    <addaction name="actionShowTextures"/>
   </widget>
   <widget class="QMenu" name="menuMesh">
    <property name="title">
     <string>Mesh</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFEM">
    <property name="title">
     <string>FEM</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionMatrices"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuMesh"/>
   <addaction name="menuModes"/>
   <addaction name="menuFEM"/>
  </widget>
  <action name="actionView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mesh overview mode</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mesh editing mode</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionMatrices">
   <property name="text">
    <string>Matrices</string>
   </property>
  </action>
  <action name="actionColourBoundary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Colour points by boundary/free status</string>
   </property>
  </action>
  <action name="actionColourValue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Colour points by value</string>
   </property>
  </action>
  <action name="actionShowMesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show only mesh</string>
   </property>
  </action>
  <action name="actionShowTextures">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show textures</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FluidFlowView</class>
   <extends>QWidget</extends>
   <header>FluidFlowView.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
